administration/src/application/components/ApplyController.tsx

```diff
@@ -57,14 +66,25 @@ const ApplyController = () => {
         application: applicationInput,
       },
     })
+
+    setState(() => ApplicationForm.initialState)
+    setFormSubmitted(true)
   }
+  const successText = `Ihr Antrag für die Ehrenamtskarte wurde erfolgreich übermittelt.
+            Sie können das Fenster schließen.`
```

Comment:
Maybe add a note, that users can return now to the app.

Reply:
I probably wouldnt do that, as the user might not come from the app (but from some other website).